docs-source/spring/content/security/azn-server/_index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Authentication and Authorization Server"
+resources:
+  - name: azn-server-arch
+    src: "azn-server-arch.png"
+    title: "Authorization Server Architecture"
+  - name: azn-stack-app-info
+    src: "azn-stack-app-info.png"
+    title: "Application Information"
+  - name: azn-passwords-console
+    src: "azn-passwords-console.png"
+    title: "AZN User passwords"
+---
+
+The Authorization Server is an engine to authenticate and authorize requests to various components in Oracle Backend for Spring Boot and Microservices. The end user can manages users using REST Endpoints.
+
+> **_NOTE:_** Oracle recommends that you change the default passwords for the default created users.
+
+## Users & Roles
+
+When deploying Oracle Backend for Spring Boot two users are created with the following roles:
+
+| User Name     | Assigned Roles        |
+|---------------|-----------------------|
+| obaas-admin   | ROLE_ADMIN, ROLE_USER |
+| obaas-user    | ROLE_USER             |
+
+All the users are stored in the database deployed when installing Oracle Backend for Spring Boot and Microservices. The Roles determines what the user is allowed to do in the environment. The allowed roles are `ROLE_ADMIN` and `ROLE_USER`.
```

Comment:
First sentence: "All users stored in the database are deployed ..."
Second sentence: "The roles determine what ..."